### PR TITLE
Add a profile for klavaro

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

File: etc/klavaro.profile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+mkdir ${HOME}/.local/share/klavaro
+mkdir ${HOME}/.config/klavaro
+whitelist ${HOME}/.local/share/klavaro
+whitelist ${HOME}/.config/klavaro
+include whitelist-common.inc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
```
Comment (SkewedZeppelin) [inline etc/klavaro.profile]: netfilter -> net none

File: etc/klavaro.profile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.local/share/klavaro
+mkdir ${HOME}/.config/klavaro
+whitelist ${HOME}/.local/share/klavaro
+whitelist ${HOME}/.config/klavaro
+include whitelist-common.inc
```
Comment (SkewedZeppelin) [inline etc/klavaro.profile]: include whitelist-var-common.inc

File: etc/klavaro.profile [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+notv
+nou2f
+novideo
+protocol unix
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-bin klavaro
```
Comment (SkewedZeppelin) [inline etc/klavaro.profile]: add tclsh
edit: more are missing: execvp: No such file or directory

File: etc/klavaro.profile [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+private-cache
+private-dev
+private-etc fonts
+private-lib
```
Comment (SkewedZeppelin) [inline etc/klavaro.profile]: remove

File: etc/klavaro.profile (hunk 3 above)
Comment (rusty-snake) [reply etc/klavaro.profile]: interesting, I don't have any problems or messages like this.
`firejail --profile=klavaro.profile --debug klavaro 2>&1 | grep execvp`
`execvp argument 0: klavaro`

Comment (SkewedZeppelin) [reply etc/klavaro.profile]: wow, you're correct
using your profile it works
but I was just testing no profile with `private-bin klavaro` and it wouldn't launch

weird

adding tclsh and bash get it going without a profile

**Comment by SkewedZeppelin** [comment]
Merged, thanks!

**Pull request merged**